lib/action_cable/channel/base.rb

```diff
@@ -73,6 +73,8 @@ class Base
       include Naming
       include Broadcasting
 
+      SUBSCRIPTION_CONFIRMATION_INTERNAL_MESSAGE = 'confirm_subscription'
```

Comment:
Wouldn't `'confirm_subscription'.freeze` inline be faster, clearer and still reap reduced allocations? :grin: